BAZProject/Networking/ServiceApi.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ServiceApi.swift
+//  BAZProject
+//
+//  Created by Daniel Alberto Vazquez Rodriguez on 27/03/23.
+//
+
+import Foundation
+
+public protocol ServiceApiProtocol : AnyObject{
```

Comment:
Sugiero seguir el [Braces](https://google.github.io/swift/#braces), la llave que abre (`{`)  no tiene espacio antes de su escritura

Asi como los dos puntos (colon `:`), se prefiere que no tenga un espacio de por medio, pero si frente a su extension.
[Type and extension declarations](https://google.github.io/swift/#type-and-extension-declarations)
```suggestion
public protocol ServiceApiProtocol: AnyObject {
```

_La misma sugerencia alrededor del codigo para mantener consistencia._